control-plane/api/mesh/v2beta1/proxy_configuration_types_test.go

```diff
@@ -109,20 +108,6 @@ func TestProxyConfiguration_MatchesConsul(t *testing.T) {
 							JsonFormat:          "jsonFormat",
 							TextFormat:          "text format.",
 						},
-						EnvoyExtensions: []*pbmesh.EnvoyExtension{
```

Comment:
This field was intentionally removed in https://github.com/hashicorp/consul/pull/19186.
I've confirmed with @ishustava that it's safe to remove this code from the test.